### DISCUSSION TITLE
[灌水]「远景爆破案」即将开庭

### DISCUSSION THREAD

**Discussion by gaojiaobei** [share121/inter-knot#952] [其他]
![微信图片_20240810194341](https://github.com/user-attachments/assets/60ee82d1-83ff-426f-8cf1-e9be14db548f)

新艾利都律法院现已发出公告，轰动一时的「远景爆破案!已确定第一次开庭时间。审理前日，包括被告(珀尔曼)、诉讼代理人(万能事务所狡兔屋)等在内的案件相关人员，将依照惯例、乘坐专用空艇前往最高裁决院候审。
据悉，本案将采用公开审理形式，共开放300个旁听席位、已经成年并具有新艾利都市民身份的人员，均可申请旁听。此外，包括丽都频道在内的8家媒体，已表示将对此案审理过程进行现场直播。

**Comment by 1451374815** [comment]
就知道这家公司有问题，只能说毫不意外

**Comment by cool-coke** [comment]
最新小道消息，被告珀尔曼好像逃跑了

**Comment by warning-Kirov-reporting** [comment]
孩子们，还是得我来

**Comment by LiHongRS** [comment]
调查局能不能有点作用？这种人都能放跑？

**Comment by sterve521** [comment]
感觉有内幕啊

**Comment by Mu-7593** [comment]
最近这么不太平，总感觉有什么大事要发生了

**Comment by Fantasy317** [comment]
听说飞艇会经过一个伴生空洞，感觉会发生一些大事

**Comment by Hongxuli2006** [comment]
我是穿越回来的，有个人会逃跑，但关键问题是我忘了他的名字，如果你能给我点好处（菲林）我倒是不介意使劲儿回想一下！

**Comment by fairy-B** [comment]
还是挺担心的QWQ，毕竟是干过那种事的公司

**Comment by hzxczxhzx** [comment]
### _**嗯。我嗅觉一丝不同寻常的味道**_
